StaticResourceSources/gsAssets/step/Find_Records_with_Global_Search.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If these are end user resources, wouldn't they go jnto the `gseuAssets` folder? See #6150